src/components/LearningCard.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import ButtonExpansion from './ButtonExpansion'
+import { useState } from 'react'
+import GlobalStyle from '../GlobalStyle'
+
+Card.propTypes = {
+  onClick: PropTypes.func,
+  isExpanded: PropTypes.bool,
+  name: PropTypes.string,
+  description: PropTypes.string,
+  tags: PropTypes.array,
+  color: PropTypes.string,
+}
+
+export default function Card({ name, description, tags, color }) {
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  return (
+    <>
+      <GlobalStyle />
+      <LearningCard isExpanded={isExpanded} backgroundColor={color}>
+        <h2 isExpanded={isExpanded}>{name}</h2>
+        {isExpanded && (
+          <>
+            <span>
+              Was ist {name}? {description}
+            </span>
+            <ul>
+              {tags.map(tag => (
+                <li key={tag}>{tag}</li>
+              ))}
+            </ul>
+          </>
+        )}
+        <ButtonExpansion
+          isExpanded={isExpanded}
+          onClick={() => setIsExpanded(!isExpanded)}
+        >
+          {isExpanded ? '-' : '+'}
+        </ButtonExpansion>
+      </LearningCard>
+    </>
+  )
+}
+
+const LearningCard = styled.div`
```

Comment:
I think this would be a `section`.
```suggestion
const LearningCard = styled.section`
```